code/modules/surgery/surgery_step.dm

```diff
@@ -98,9 +98,9 @@
 		modded_time = time
 
 	var/was_sleeping = (target.stat != DEAD && target.IsSleeping())
-
-	if(do_after(user, modded_time, target = target, interaction_key = user.has_status_effect(/datum/status_effect/hippocratic_oath) ? target : DOAFTER_SOURCE_SURGERY)) //If we have the hippocratic oath, we can perform one surgery on each target, otherwise we can only do one surgery in total.
```

Comment:
This isn't related to anything voidcrew related and seems like a balance, can we revert this?